src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+const Books = () => {
+  const booksArray = [{
+    id: 1, title: 'Dune', author: 'Frank Herbert', genre: 'Science Fiction',
+  }];
+  return (
+    <>
+      <Navbar />
```

Comment:
- [ ] Instead of having `<Navbar />` here and also in the **Categories.js** file thereby breaking the DRY rule, I would recommend you remove it from here and in **Categories.js** and move it to **App.js** where you have your routes configured

  ```
  // In App.js

  <Navbar />
  <Routes>
    ...
  ```

Reply:
I have implemented this change.